examples/todo/imports/api/resolvers.js

```diff
@@ -32,6 +32,10 @@ export const resolvers = {
     async removeCheckedTasks() {
       TasksCollection.removeChecked();
     },
+    async editProfile(root, { user }, { userId }) {
+      Users.update(user._id, { $set: { ...user } });
```

Comment:
@tgmarinho if only the logged user can update his own profile it's better to use `userId` on the update selector too, it's just for safety

Reply:
Yes, We changed it on Saturday